bin/physlr-make

```diff
@@ -169,11 +168,13 @@ help:
 	@echo " 			1: periodic messages."
 	@echo " 			2: progress bar."
 	@echo " 			3: verbose messages."
-	@echo "	minimizer_overlap		% of edges to remove in the overlap graph [90]. Available options are stLFR, 10X, or [0-100]."
-	@echo "					stLFR: 85."
-	@echo "					10X: 92.5."
-	@echo "					Default values for stLFR and 10X were obtained by finding a value that worked well across multiple data sets."
+	@echo "	m 	% of edges to remove in the overlap graph [90]. Available options are [0-100]."
+	@echo "	protocol		Available options are stlfr and 10x. Specify the protocol option to use the default m value for the respective linked read protocol."
```

Comment:
Are we going to use all lowercase or mixed upper-case/lowercase? I know Shaun suggested using `stlfr` all lower-case, and it could make sense to make that change in this PR.
Either way, we need to be consistent.

Reply:
oh i forgot to change the examples